x/crosschain/keeper/msg_server_update_tss.go

```diff
@@ -70,3 +72,9 @@ func (k msgServer) UpdateTssAddress(
 
 	return &types.MsgUpdateTssAddressResponse{}, nil
 }
+
+// GetChainsSupportingMigration returns the chains that support migration.
```

Comment:
Can we leave more details in the comments? Why specifically filtering ethereum and bitcoin?

Reply:
I am not sure what details to add, bitcoin and Ethereum, consensus  are the only ones that support TSS migration right now

Reply:
e32c2a87997922c99ad8a9e7709df90d10050b69